src/pages/authorization/authorization-in-practice/index.mdx

```diff
@@ -36,31 +36,40 @@ To use any method from the Agent Chat, Customer Chat, and Configuration API, you
 
 <img
   alt="LiveChat Developer Console"
-  src="/images/authorization/livechat-authorization-block-2x.png"
-  width="800px"
-  height="458px"
+  src="/images/authorization/livechat-authorization-in-practice-building-block.png"
```

Comment:
I have changed the screens and adjusted some info/text :)
I would appreciate it if you could check if any of the sensitive info is not visible. I was making sure if I had hidden the same info as in the original screens. However, double-check would be really appreciated :)